packages/eas-cli/src/commands/update/index.ts

```diff
@@ -733,3 +735,11 @@ async function checkEASUpdateURLIsSetAsync(exp: ExpoConfig): Promise<void> {
     );
   }
 }
+
+export const truncatePublishUpdateMessage = (originalMessage: string | undefined = ''): string => {
+  if (originalMessage.length > 1024) {
+    Log.warn('Update message exceeds the allowed charater limit. Truncating message...');
```

Comment:
I would say what the limit is.

Reply:
i wasn't sure if we wanted to be transparent about it

Reply:
'charater' -> 'character'